CMakeLists.txt

```diff
@@ -94,6 +115,10 @@ else()
   message(STATUS "Building StableHLO embedded in another project")
 endif()
 
+if(LLVM_ENABLE_ZLIB)
+  find_package(ZLIB)
```

Comment:
Curious on error checking. We have special handling for `find_program` but not `find_package`.

Looks like we can add `find_package(... REQUIRED)` and it will error for us if it cant be found. Same for `find_program` in CMake 3.18+

Reply:
For now, let's keep this as is to replicate the MLIR-HLO build behavior (even minor divergence may complicate migrations). Can revisit in the future.